_news/2025-05-23-sen-symposium.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: news-item
+date: 2025-05-23
+title: "AI4SE showcases research at the 11th SEN Symposium 2025"
+tracks: [3,4] 
+---
+
+The 11ᵗʰ Dutch National Symposium on Software Engineering (SEN 2025) took place in Amsterdam. The symposium, organised by VERSEN, gathered researchers from across the Netherlands for a full day of keynotes, invited talks, and lightning talks.
+
+**AI4SE highlights from the official program:**
+
+* **Invited talk** — Burcu Kulahcioglu Ozkan (TU Delft / AI4SE Track 4): Model‑guided Testing of Distributed Systems.
+
+* **Session chair** — Dr Maliheh Izadi (TU Delft / AI4SE Track 1): Chaired the afternoon session.
+
+* **Host‑assistant** — Roham Koohestani (AI4SE Track 3 / Research Assistant / JetBrains Intern): Assisted during the afternoon session.
```

Comment:
You can also add ziyou, no need to include their full roles, just adding "students in track 3" suffices.